projects/1.16/assets/into-the-omega/intotheomega/lang/zh_cn.json

```
@@ -0,0 +1,18 @@
+{
+  "itemGroup.intotheomega.group": "Into the Ω",
+  "item.intotheomega.omega_crystal": "Ω水晶",
+  "item.intotheomega.omega_enchanted_book": "附魔书",
+  "item.intotheomega.omega_sword": "Omega剑",
+  "item.intotheomega.omega_pickaxe": "Omega镐",
+  "item.intotheomega.omega_hoe": "Omega锄",
+  "item.intotheomega.omega_shovel": "Omega铲",
+  "item.intotheomega.omega_axe": "Omega斧",
+  "item.intotheomega.omega_helmet": "Omega头盔",
+  "item.intotheomega.omega_chestplate": "Omega胸甲",
+  "item.intotheomega.omega_leggings": "Omega护腿",
+  "item.intotheomega.omega_boots": "Omega靴子",
+  "item.wolveswitharmor.omega_wolf_armor": "§dOmega狼盔甲",
```

```suggestion
  "item.wolveswitharmor.omega_wolf_armor": "§dOmega狼铠",
```
https://github.com/Draylar/wolves-with-armor/blob/1378a06b4e804ea57654920e74585e388cd2737d/src/main/resources/assets/wolveswitharmor/lang/zh_cn.json#L2